modules/test/playwright/tests/portal-workflow-kaleo-designer-web/assignments.spec.ts

```diff
@@ -74,3 +74,32 @@ test('cannot see Scripted Assignment option when script management configuration
 		await page.$(`#assignment-type option[value="scriptedAssignment"]`)
 	).toBeNull();
 });
+
+test('role name search box does not support regex strings', async ({
+	diagramViewPage,
+	nodePropertiesSidebarPage,
+	page,
+	processBuilderPage,
+}) => {
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.clickNode('review');
+
+	await nodePropertiesSidebarPage.editAssignmentButton.click();
+
+	await page.locator('input#role-name').first().fill('a');
```

Comment:
Hey @devpedrotavares, I noticed that you use this locator twice in your test. Can you try to set it on the page it belongs to?

Also, try more semantic approaches, I believe `getByLabel` should work in this case. [see](https://playwright.dev/docs/locators#quick-guide)

Reply:
Done!